glossary.md

```diff
@@ -11,6 +11,7 @@ Just a list of acronyms I've run across in the Ruby source code and their meanin
 | `ec` | Execution Context. The top level VM context, points at the current `cfp` |
 | `ep` | Environment Pointer. Local variables, including method parameters are stored in the `ep` array. The `ep` is pointed to by the `cfp` |
 | `iseq` | Instruction Sequence.  Usually "iseq" in the C code will refer to an `rb_iseq_t` object that holds a reference to the actual instruction sequences which are executed by the VM. The object also holds information about the code, like the method name associated with the code. |
+| `lep` | Local Environment Pointer. |
```

Comment:
Yes! This is a good addition. I elaborated a bit

```suggestion
| `lep` | Local Environment Pointer. An `ep` which is tagged `VM_ENV_FLAG_LOCAL`. Usually this is the `ep` of a method (rather than a block, whose `ep` isn't "local") |
```